client/app/services/authentication/authentication.service.js

```diff
@@ -21,6 +21,12 @@ class AuthenticationService {
       .catch((res) => this.$q.reject(res.data));
   }
 
+  verify(userEmail) {
+    return this.$http.post("/api/users/resend_verification/", { email: userEmail })
+      .then((res) => res.data)
+      .then(() => this.data = "Email Sent");
```

Comment:
Is this used? I wouldn't set (translatable) messages in this backend-facing service, but instead in the controller.